src/domain/controller/LunchRecommendation.js

```diff
@@ -0,0 +1,134 @@
+import Restaurant from '../model/Restaurant';
+import RestaurantList from '../model/RestaurantList';
+import { $$$ } from '../../utils';
+import webView from '../../view/webView';
+import { DEFAULT_RESTAURANTS } from '../../constants';
+
+class LunchRecommendation {
+  #restaurants = new RestaurantList();
+
+  constructor() {
+    DEFAULT_RESTAURANTS.forEach((restaurant) => {
+      const defaultRestaurant = new Restaurant(restaurant);
+      this.#restaurants.add(defaultRestaurant);
+    });
+
+    const userList = JSON.parse(localStorage.getItem('userList'));
+
+    if (userList) {
+      userList.forEach((restaurant) => {
+        const objectToClass = new Restaurant(restaurant);
+        this.#restaurants.add(objectToClass);
+      });
+    }
```

Comment:
```suggestion
    const userList = JSON.parse(localStorage.getItem('userList')) ?? [];

    userList.forEach((restaurant) => {
      const objectToClass = new Restaurant(restaurant);
      this.#restaurants.add(objectToClass);
    });
```


(조건)문보다는 표현식을 사용하는게 더 좋습니다 😄

Reply:
좋은 방법 알려주셔서 감사합니다 :+1::+1: